test-runner/action.yml

```diff
@@ -56,7 +56,7 @@ runs:
           python -m pip install -r mlops/test-runner/requirements.txt
     - name: Test
       shell: bash
-      run: venv/bin/python mlops/test-runner/test_runner.py -f ${{ inputs.test_dir }}/tests.yaml -l ${{ inputs.test_dir }}/logs -a ${{ inputs.recipe_dir }}/.actions.json -v
```

Comment:
This MLOps dir is important for other repos to use this composite

Reply:
Will make the changes after I have done revalidating.